### PR TITLE
Update member.php

### PR DESCRIPTION
**Pull request by deepend-tildeclub**
shows depreciation notice on main page when signed out.

### CODE REVIEW

File: src/classes/member.php [hunk 1]
```diff
@@ -47,8 +47,12 @@ function select($memberID, $numericIDOnly = true) {
...
 			}
 		} else {
-			$memberID = $this->MySQL->real_escape_string($memberID);
-			$result = $this->MySQL->query("SELECT * FROM ".$this->strTableName." WHERE username = '$memberID'");
+			if ($memberID !== null) {
+				$memberID = $this->MySQL->real_escape_string($memberID);
+				$result = $this->MySQL->query("SELECT * FROM ".$this->strTableName." WHERE username = '$memberID'");
+			} else {
+				return false;
+			}
```
Comment (RedDragonWebDesign) [inline src/classes/member.php:55]: Better to use a [guard clause](https://deviq.com/design-patterns/guard-clause) here. Makes code more readable:

```php
if ($memberID === null) {
	return false;
}

$memberID = $this->MySQL->real_escape_string($memberID);
$result = $this->MySQL->query("SELECT * FROM ".$this->strTableName." WHERE username = '$memberID'");
```

**Review by RedDragonWebDesign — CHANGES_REQUESTED** [review]

**Comment by RedDragonWebDesign** [comment]
Is the warning on http://localhost/src/index.php? I don't see it.

**Comment by deepend-tildeclub** [comment]
> Is the warning on http://localhost/src/index.php? I don't see it.

Kinda frustrating.    I'm not getting the error anymore either.  Wonder if something else being fixed helped sort this issue out somehow.

**Comment by RedDragonWebDesign** [comment]
Sounds good. Will close for now. Can always reopen if the error comes back.

**Pull request closed without merging**